manifests/init.pp

```diff
@@ -99,10 +108,12 @@
   Boolean $fwd_conntrack = false,
   Boolean $inet_filter = true,
   Boolean $nat = true,
+  Boolean $allow_unmanaged_rules = true,
```

Comment:
`Boolean $purge_unmanaged_rules = false` is clearer to me as variable name as to what it will do. `allow_unmanaged_rules` sounds like blocking  rules from  even being added in the first place.

Reply:
Yup, it's probably better. I'll change it.

Reply:
It's done.